Added CLI Command run with PHPUnit 10

manishranjan-adobe opened the pull request:
## Purpose of this pull request

This pull request (PR) fixes the Config file allure/allure.config.php doesn't exist issue.

run command:
./vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist

(doc: https://developer.adobe.com/commerce/testing/guide/unit/command-line/)

Error: Config file allure/allure.config.php doesn't exist

[GitHub Issue Ref] : (https://github.com/magento/magento2/issues/36702#issuecomment-2358670590)


## Affected pages

<!-- REQUIRED List the affected pages on developer.adobe.com (URLs). Not necessary for large numbers of files. -->

- ...

## Links to Magento Open Source code

<!--  OPTIONAL - REMOVE THIS SECTION IF NOT USED. If this pull request references a file in a Magento Open Source or Adobe Commerce codebase repository, add it here. -->

- ...

<!--
If you are fixing a GitHub issue, using the GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) closes the issue when this pull request is merged. Example: `Fixes #1234`.
`main` is the default branch. Merged pull requests to `main` go live on the site automatically. Any requested changes to content on the `main` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.
See Contribution guidelines (https://github.com/AdobeDocs/commerce-testing/blob/main/.github/CONTRIBUTING.md) for more information.
-->

jeff-matthews commented on src/pages/guide/unit/command-line.md:
```diff
@@ -89,3 +89,21 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+
+If you encounter an error similar to `Bootstrapping of extension Qameta\Allure\PHPUnit\AllureExtension failed: Config file allure/allure.config.php doesn't exist`, follow the steps below to resolve it and run your unit tests successfully.
+
+#### Steps to Run Tests via CLI:
```
```suggestion
```

jeff-matthews commented on src/pages/guide/unit/command-line.md:
```diff
@@ -89,3 +89,21 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+
+#### Steps to Run Tests via CLI:
+
+1. Navigate to the unit test directory
```
```suggestion
1. From the project root, change into the unit test directory:

```

jeff-matthews commented on src/pages/guide/unit/command-line.md:
```diff
@@ -89,3 +89,21 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+#### Steps to Run Tests via CLI:
+
+1. Navigate to the unit test directory
+   From the project root, execute the following command to change into the unit test directory:
```
```suggestion
```

jeff-matthews commented on src/pages/guide/unit/command-line.md:
````diff
@@ -89,3 +89,21 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+   ```
+
+2. Run PHPUnit
+   Use the following command to execute PHPUnit with the appropriate configuration:
````
```suggestion
```

jeff-matthews commented on src/pages/guide/unit/command-line.md:
````diff
@@ -89,3 +89,21 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+   cd dev/tests/unit/
+   ```
+
+2. Run PHPUnit
````
```suggestion
2. Run PHPUnit with the appropriate configuration.
```

jeff-matthews reviewed (commented):
Thanks @manishranjan-adobe. I just have a few minor editorial suggestions to comply with doc style/standards.

manishranjan-adobe commented:
@jeff-matthews I've addressed the suggestions you provided in the pull request. Could you kindly review it again at your earliest convenience? Let me know if there's anything else you'd like me to adjust.

jeff-matthews commented on src/pages/guide/unit/command-line.md:
```diff
@@ -89,3 +89,17 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+
+If you encounter an error similar to `Bootstrapping of extension Qameta\Allure\PHPUnit\AllureExtension failed: Config file allure/allure.config.php doesn't exist`, follow the steps below to resolve it and run your unit tests successfully.
+
+1. From the project root, change into the unit test directory
```
```suggestion
1. From the project root, change into the unit test directory.

```

jeff-matthews commented on src/pages/guide/unit/command-line.md:
````diff
@@ -89,3 +89,17 @@ Add the following line in the `<php>` block to disable the PHP memory limit duri
+   cd dev/tests/unit/
+   ```
+
+2. Run PHPUnit with the appropriate configuration.
````
```suggestion
2. Run PHPUnit with the appropriate configuration.

```

jeff-matthews reviewed (commented):
Thanks @manishranjan-adobe. I made a couple more small suggestions. The linter will fail if fenced code blocks are not surrounded by blank lines.

Also, please sign the Adobe CLA in the checks section.

manishranjan-adobe commented:
@jeff-matthews I've implemented the follow-up fixes you suggested.
